airbyte-scheduler/src/main/java/io/airbyte/scheduler/temporal/TemporalPool.java

```diff
@@ -67,4 +79,36 @@ public void run() {
     factory.start();
   }
 
+  private static void waitForTemporalServerAndLog() {
+    LOGGER.info("Waiting for temporal server...");
+
+    while (!getNamespaces().contains("default")) {
+      LOGGER.warn("Waiting for default namespace to be initialized in temporal...");
+      wait(5);
```

Comment:
could we do 2 instead of 5? i feel like increments of 5 seconds is a long time given current startup speeds.